.github/workflows/deployment.yml

```diff
@@ -156,3 +153,67 @@ jobs:
           credentials: ${{ secrets.AZURE_CREDENTIALS }}
           acr: ${{ secrets.ACR_PASSWORD }}
           functions: ${{ matrix.functions }}
+
+  # 5. Notification
+  notify:
+    name: Notification 🔔
+    needs: [setup, database, webapp, functions]
+    environment: ${{ needs.setup.outputs.environment }}
+    runs-on: [self-hosted, DTFS, deployment]
+
+    env:
+      ENVIRONMENT: ${{ needs.setup.outputs.environment }}
+      WEBHOOK: ${{ secrets.MSTEAMS_WEBHOOK }}
+
+    steps:
+      - name: Facts
+        run: |
+
+      - name: MS Teams
+        run: |
+          curl --location '${{ env.WEBHOOK }}' \
+          --header 'Content-Type: application/json' \
+          --data-raw '{
+          "@type": "MessageCard",
+          "@context": "http://schema.org/extensions",
+          "themeColor": "00703c",
+          "title": "🚀 main : ${{ env.ENVIRONMENT }}",
+          "summary": "Deployed '\''main'\'' branch to '\''${{ env.ENVIRONMENT }}'\'' DTFS environment",
```

Comment:
should the branch name come from github? what if deploying release?

Reply:
In this case it is always deploying from `main`.